functions/bouncer/src/middleware/stripeEventParse.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console */
+import debug from 'debug';
+import stripelib from 'stripe';
+import * as Sentry from '@sentry/node';
+import envConfig from '../envConfig';
+
+const dlog = debug('that:api:functions:bouncer:stripeParseMw');
+const stripe = stripelib(envConfig.stripe.apiKey);
+
+export default function stripeEventParse(req, res, next) {
+  dlog('stripe event parser called');
+
+  const whRes = {
+    bouncer: true,
+    isQueued: false,
+    isValid: false,
+    stages: ['stripeEventParse'],
+  };
+
+  const sig = req.headers['stripe-signature'];
```

Comment:
and this a constant?

Reply:
Constant? I am not following. They are what they are for that file.
It is a way we track which middleware has been visited to aid in troubleshooting. This data will be in Sentry.